pandas/core/arrays/datetimes.py

```diff
@@ -1704,6 +1707,7 @@ def sequence_to_dt64ns(data, dtype=None, copy=False,
             data, inferred_tz = objects_to_datetime64ns(
                 data, dayfirst=dayfirst, yearfirst=yearfirst)
             tz = maybe_infer_tz(tz, inferred_tz)
+            int_as_wall_time = False
```

Comment:
IIRC this was for a mixed tz test. Verifying now, and will add a comment..

Reply:
Sorry, not necessarily mixed. Just cases where a sequence of Timestamp objects w/ tz are passed.

```python
result = Index([Timestamp('2011-01-01 10:00', tz='Asia/Tokyo')])
```

that seems to extract the tz and extract the i8 `.value` from the timestamps.